CorsixTH/Lua/queue.lua

```diff
@@ -313,16 +313,22 @@ function Queue:movePatient(index, new_index)
 end
 
 --! Called when reception desk is destroyed, or when a room is destroyed from a crashed machine.
-function Queue:rerouteAllPatients(action)
+--!param room_id Id of the room to reroute to, 'nil' for reception.
+function Queue:rerouteAllPatients(room_id)
   for _, humanoid in ipairs(self) do
     -- check by class type as staff/vips shouldn't get a SeekRoomAction
     if class.is(humanoid, Patient) then
```

Comment:
Not you, but note to self we will need to reroute the vip if he was going to reception, or his vip-ing doesn't begin (I've documented this somewhere else so no need for action here.)